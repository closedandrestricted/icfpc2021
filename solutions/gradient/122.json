{"vertices":[[185,313],[146,275],[193,292],[208,216],[157,222],[173,237],[173,247],[221,267],[164,261],[170,254],[191,200],[174,198],[173,264],[143,221],[122,176],[207,230],[158,225],[181,287],[209,271],[191,238],[211,173],[176,237],[151,245],[212,273],[191,236],[238,249],[206,218],[162,291],[230,193],[189,251],[192,260],[168,189],[210,198],[186,220],[144,285],[213,218],[206,231],[218,250],[219,215],[191,200],[167,304],[225,279],[170,189],[224,285],[249,209],[235,213],[190,248],[203,247],[193,223],[235,230],[213,238],[237,219],[159,288],[153,256],[213,189],[203,227],[215,206],[173,274],[196,275],[196,189],[153,224],[213,231],[186,216],[210,306],[192,214],[193,298],[218,239],[192,219],[189,188],[246,250],[183,220],[210,198],[260,260],[216,205],[193,240],[160,224],[172,213],[174,239],[204,229],[194,237],[186,214],[237,228],[229,239],[187,240],[211,268],[244,225],[212,234],[149,259],[194,208],[169,255],[196,195],[175,197],[241,244],[225,203],[192,215],[197,190],[217,240],[185,237],[218,222],[231,227],[269,243],[189,253],[168,259],[211,177],[244,240],[214,206],[182,250],[167,226],[220,254],[214,229],[179,212],[219,253],[273,274],[179,267],[172,171],[231,258],[167,216],[217,223],[180,254],[168,234],[186,247],[237,263],[178,214],[182,221],[169,261],[182,195],[157,198],[212,247],[218,239],[194,268],[195,228],[209,274],[191,220],[189,233],[192,244],[155,209],[183,226],[195,203],[160,204],[211,189],[203,245],[181,239],[166,256],[192,236],[148,274],[179,283],[215,222],[217,219],[215,207],[186,180],[179,270],[216,193],[182,213],[202,201],[196,288],[197,296],[189,297],[184,260],[221,214],[214,191],[170,274],[204,218],[196,212],[167,281],[161,259],[167,248],[155,288],[183,207],[205,223],[130,280],[217,201],[157,290],[198,236],[219,279],[205,261],[204,218],[183,279],[187,233],[187,177],[212,271],[201,202],[197,241],[193,245],[123,271],[218,205],[183,286],[185,247],[200,259],[188,274],[148,257],[207,217],[221,239],[194,287],[207,262],[191,265],[177,251],[195,266],[158,287],[173,264],[236,256],[176,303],[189,233],[176,247],[232,269],[160,223],[162,293],[211,261],[191,233],[189,214],[219,231],[207,209],[175,270],[174,251],[233,263],[190,272]]}