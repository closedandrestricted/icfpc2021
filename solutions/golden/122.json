{"vertices":[[50,105],[93,77],[352,293],[305,197],[298,256],[286,234],[328,250],[63,131],[77,177],[345,338],[258,264],[267,198],[101,100],[147,72],[109,102],[38,162],[113,187],[154,93],[314,294],[112,171],[91,181],[241,228],[223,209],[152,128],[293,262],[138,97],[125,193],[307,345],[119,176],[136,201],[250,153],[157,111],[106,160],[141,189],[182,116],[141,163],[127,137],[270,241],[176,92],[137,173],[210,205],[246,224],[192,203],[209,149],[84,157],[126,179],[170,157],[160,123],[157,184],[184,138],[136,173],[122,184],[252,176],[202,138],[167,154],[205,163],[208,103],[192,231],[249,216],[193,177],[188,168],[144,200],[166,186],[208,178],[203,193],[209,204],[215,196],[122,198],[188,171],[215,190],[202,177],[169,222],[204,172],[137,175],[141,216],[175,212],[189,208],[191,197],[200,197],[204,142],[207,196],[196,186],[137,233],[212,205],[175,198],[167,178],[155,230],[178,209],[203,185],[237,163],[115,173],[135,172],[150,230],[193,168],[181,200],[159,187],[242,178],[216,204],[159,241],[146,249],[178,216],[215,219],[233,222],[257,250],[169,210],[231,234],[208,189],[199,217],[135,222],[180,218],[194,252],[132,225],[152,198],[217,174],[182,166],[119,237],[202,146],[178,239],[202,187],[185,279],[219,195],[145,235],[243,199],[197,189],[197,302],[259,215],[205,188],[198,223],[190,297],[105,216],[173,226],[217,154],[224,298],[222,311],[159,248],[245,346],[240,201],[187,268],[165,256],[177,286],[162,271],[173,292],[167,270],[149,271],[187,152],[175,295],[201,249],[272,174],[208,269],[207,197],[171,289],[158,322],[185,268],[158,324],[120,272],[154,295],[221,123],[185,274],[143,306],[253,320],[191,277],[187,314],[199,272],[158,279],[81,296],[256,308],[151,319],[226,298],[160,302],[163,342],[162,336],[161,370],[193,296],[175,314],[170,292],[182,343],[196,307],[166,322],[189,307],[150,295],[164,320],[154,318],[47,308],[206,319],[181,324],[142,345],[156,332],[196,330],[72,326],[135,348],[175,309],[163,266],[194,298],[167,308],[191,320],[147,333],[129,319],[215,325],[137,321],[180,281],[192,321],[111,318],[170,331],[144,297],[99,345],[206,363],[125,284],[103,311],[112,372],[97,340],[122,332],[89,271],[126,411],[73,365],[102,400]]}